src/pages/docs/runbooks/config-as-code-runbooks.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-11-24
+modDate: 2024-11-24
+title: Configuration as Code runbooks
+description: Details about using configuration as code with runbooks.
+navOrder: 20
+icon: fa-brands fa-git-alt
+---
+
```

Comment:
Depending on how far through we are with any rollout when these docs are merged we might want some sort of "Support for CaC runbooks is rolling out to Octopus Cloud now and will be available in Octopus Server in a future release".